src/content/configuration/index.mdx

```diff
@@ -43,1130 +44,48 @@ If for some reason you want to use a different configuration file depending on c
-    recordsInputPath: path.resolve(__dirname, "build/records.json"),
-    recordsOutputPath: path.resolve(__dirname, "build/records.json"),
-    // store ids into a file to make the build even more deterministic
-  </advanced>
-  <advancedCaching "#">
-    <default>
-      /* Advanced caching configuration (click to show) */
-    </default>
-    cache: false, // boolean
-    // disable/enable caching
-    snapshot: {
-      managedPaths: [ path.resolve(__dirname, "node_modules") ],
-      // paths that are snapshot using only package.json name and version
-      immutablePaths: [ path.resolve(__dirname, ".yarn/cache") ],
-      // paths that doesn't need to be snapshot as they are immutable
-      module: { timestamp: true, hash: true },
-      resolve: { timestamp: true, hash: false },
-      resolveBuildDependencies: { timestamp: true, hash: false },
-      buildDependencies: { timestamp: true, hash: true },
-      // snapshot method for different operations
-    },
-    watch: true, // boolean
-    // enables watching
-    watchOptions: {
-      aggregateTimeout: 1000, // in ms
-      // aggregates multiple changes to a single rebuild
-      poll: true,
-      poll: 500, // interval in ms
-      // enables polling mode for watching
-      // must be used on filesystems that doesn't notify on change
-      // i. e. nfs shares
-    },
-  </advancedCaching>
-  <advancedBuild "#">
-    <default>
-      /* Advanced build configuration (click to show) */
-    </default>
-    infrastructureLogging: {
-      level: "none",
-      level: "error",
-      level: "warn",
-      level: "info", // (default)
-      level: "log",
-      level: "verbose",
-      debug: true,
-      debug: /webpack/,
-      debug: [ "MyPlugin", /webpack/ ]
-    },
-    parallelism: 1, // number
-    // limit the number of parallel processed modules
-    profile: true, // boolean
-    // capture timing information
-    bail: true, //boolean
-    // fail out on the first error instead of tolerating it.
-    dependencies: ["name"],
-    // When using an array of configs this can be used to reference other
-    // configs and let this config run after the other config at initial build
-  </advancedBuild>
-}
-```
+## Set up a new webpack project
 
 W> webpack applies configuration defaults after [plugins defaults](/contribute/writing-a-plugin/#configuration-defaults) are applied.
 
-Use [webpack-cli's `init` command](/api/cli/#init) to rapidly generate webpack configuration files for your project requirements, it will ask you a couple of questions before creating a configuration file.
+Webpack has a huge set of options which might be overwhelming to you, please take advantage of [webpack-cli's `init` command](/api/cli/#init) which could rapidly generate webpack configuration files for your project requirements, it will ask you a couple of questions before creating a configuration file.
 
 ```bash
-npx webpack-cli init
+$ npx webpack init
```

Comment:
```suggestion
npx webpack init
```
I feel like removing the $ would be better. It'll be a little hard to copy command?

Reply:
I don't think it matters if you're going to select the texts manually. But I do consider introducing a copy button in near future for those code blocks.

Reply:
In case we consider to include copy button it is better to remove these $ prefixes. Considering no other documentation site have them ex: https://beta.reactjs.org/learn/start-a-new-react-project, eslint.org

Reply:
It should be easy to remove `$` in event handler if we want to. But I do agree users might copy `$` without knowing its intention here. And in the case of multiple-line commands, `$` will cause more trouble. Thanks for your input.